cmd/object-api-interface.go

```diff
@@ -139,4 +140,7 @@ type ObjectLayer interface {
 	PutObjectTag(context.Context, string, string, string) error
 	GetObjectTag(context.Context, string, string) (tagging.Tagging, error)
 	DeleteObjectTag(context.Context, string, string) error
+
+	// OBD
+	GetRWLock(context.Context, string, time.Duration, time.Duration) (RWLocker, error)
```

Comment:
Just for OBD lets not add new object layer locking functions NewNSLock is already exposed at Object layer which can be used.